src/Translator/src/DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,14 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->children()
                 ->scalarNode('dump_directory')->defaultValue('%kernel.project_dir%/var/translations')->end()
+                ->arrayNode('excluded_domains')
+                    ->scalarPrototype()->end()
```

Comment:
Could you add a tiny comment/doc there (improve DX when debug config) ?

(for both)